server/api.ts

```diff
@@ -1059,6 +1059,20 @@ router.getAsync("/stage-stats", async (req, res) => {
   res.send(stageStats);
 });
 
+/**
+ * POST /api/stage-stats
+ * Edit stats for a tournament stage
+ * Params:
+ *   - stats: updated StageStats object
+ */
+router.postAsync("/stage-stats", ensure.isAdmin, async (req, res) => {
```

Comment:
as part of the ts refactor, if would be cool if you could migrate these stats routes to use typescript definitions for req/res and move the routes into some `server/api/stats.py` module
(see 4eda2710265f252b0083818f4bec823c5853eeea for an example of this)

could be done in a separate PR tho if you prefer

Reply:
let's do separate PR :eh: